test/functional-tests/specs/browserActions.spec

```diff
@@ -6,7 +6,7 @@
 * Navigate to "http://localhost:3001/"
 * Click "Multiple Windows"
 * Assert page navigated to "/windows"
-* Click "Click Here"
+* Click "Click Here" with options "{\"navigationTimeout\": 40000,\"waitForNavigation\": true, \"waitForEvents\": [\"firstMeaningfulPaint\", \"DOMContentLoaded\"]}"
```

Comment:
we may not need this

Reply:
Make sense after [this](https://github.com/getgauge/taiko/blob/96b369d0533d539ef6556f9d761bad22fc7f0202/lib/doActionAwaitingNavigation.js#L110) change we don't need this